packages/genesys-spark-components/src/components/beta/gux-segmented-control/gux-segmented-control-item/gux-segmented-control-item.scss

```diff
@@ -30,40 +31,26 @@ button {
 
 .gux-container {
   height: var(--gse-ui-segmentedControl-height);
-  border-color: #a3a7b0; //TODO https://inindca.atlassian.net/browse/COMUI-2294
-  border-style: var(
-    --gse-ui-segmentedControl-border-style
-  ); //TODO https://inindca.atlassian.net/browse/COMUI-2294
-
-  border-width: var(
-    --gse-ui-segmentedControl-border-width
-  ); //TODO https://inindca.atlassian.net/browse/COMUI-2294
-
+  border-color: var(--gse-ui-segmentedControl-divider-color);
+  border-style: var(--gse-ui-segmentedControl-divider-style);
+  border-width: var(--gse-ui-segmentedControl-divider-width);
```

Comment:
Minor but would `border:var(--gse-ui-segmentedControl-divider-width) var(--gse-ui-segmentedControl-divider-style) var(--gse-ui-segmentedControl-divider-width)` be better here.

Reply:
Because of the long token names I have been finding the shorthand `border` property looks a bit messy :).

Reply:
That is true tbf  😄